eclair-node/src/main/scala/fr/acinq/eclair/gui/Handlers.scala

```diff
@@ -64,11 +65,11 @@ class Handlers(setup: Setup) extends Logging {
     }
   }
 
-  def getPaymentRequest(amountMsat: Long, textarea: TextArea) = {
-    (paymentHandler ? 'genh).mapTo[BinaryData].map { h =>
-      Platform.runLater(new Runnable() {
+  def getPaymentRequest(amountMsat: MilliSatoshi, textarea: TextArea) = {
```

Comment:
`getPaymentRequest` -> `receive` would be more consistent with the `send` counterpart